external-crates/move/tools/move-package/src/package_lock.rs

```diff
@@ -4,15 +4,21 @@
 
 use named_lock::{NamedLock, NamedLockGuard};
 use once_cell::sync::Lazy;
+use sha2::{Digest, Sha256};
```

Comment:
We typically invoke hash functions from fastcrypto only to avoid introducing multi hash deps. Also Sui's default is blake2b_256 (3x faster than sha3). See DefaultHash https://github.com/MystenLabs/sui/blob/main/crates/sui-types/src/crypto.rs#L76

Reply:
@kchalkias thanks for the review! Totally agreed that everything should go through fast-crypto on the Sui side/use blake2b. Issue here is this lives outside of Sui (it's on the Move side of things) where we're already using these same crypto functions (Sha256). So not sure what the best thing to do here.